code/datums/diseases/japanesefever.dm

```diff
@@ -0,0 +1,92 @@
+				spawn(100)
+					to_chat(affected_mob, "<span class='warning'>Clarity hits you like a punch to the gut. You want to rip your hair out, tear apart all these damned clothes and never look at anything pink again. It's disgusting. DISGUSTING.</span>")
+					spawn(100)
+						to_chat(affected_mob, "<span class='warning'>How - no, why did it end up like this? Only the most miserable creature could invent such brain rotting material like anime. You can almost see his sadonic smirk in your head. You want to kill him.</span>")
+						spawn(100)
+							to_chat(affected_mob, "<span class='notice'>A gentle flushing sensation spreads throughout your body.</span>")
+							spawn(30)
+								to_chat(affected_mob, "<span class='notice'>Oh... that feels good...</span>")
+								spawn(60)
+									to_chat(affected_mob, "<span class='notice'>You can't WAIT to be a cute girl for everybody, nya!.</span>")
+									revelation_had = 2
+		else
+			return
+			
+/datum/disease/japanesefever/AffectSpeech(var/datum/speech/speech)
+	var/message = speech.message
+	var/listA = list("cute","adorable","cuddly")
+	for(var/word in listA)
+		message = replacetext(message, word, "kawaii~")
+
+	var/listB = list("idiot","dumbass","dummy","retard","faggot","nigger","spic","motherfucker","fucker","dumbshit","dipshit","dumbfuck","autist","savant","troglodyte")
+	for(var/word in listB)
+		message = replacetext(message, word, pick("BAKA","meanie"))
+		
+	var/listC = list("heya","hello","greetings")
+	for(var/word in listC)
+		message = replacetext(message, word, "ohaiyo~")
+
+	var/listD = list("tajaran","catbeast")
+	for(var/word in listD)
+		message = replacetext(message, word, "catfriend")
+
+	var/listE = list("ugh","fuck","sigh","damnit","damn","shit")
+	for(var/word in listE)
+		message = replacetext(message, word, "uwa~")
+		
+	var/listF = list("honestly", "to be honest")
+	for(var/word in listF)
+		message = replacetext(message, word, "desu~")
+		
+	var/listG = list("captain","cap","cappy","boss","hos","head of security","head of shitcurity","hop","head of personnel","head of personell","head of personnell","rd","research director","cmo","chief medical officer","qm","quartermaster")
+	for(var/word in listG)
+		message = replacetext(message, word, pick("sempai", "senpai"))
+		
+	message = replacetext(message, "sorry", "gomen")
+	message = replacetext(message, "traitor", "real big meanie")
+	message = replacetext(message, "syndicate", "super big meanies")
+	message = replacetext(message, "vampire", "b-blood s-sucker")
+	message = replacetext(message, "changeling", "b-body ab-bsorber a-ack")
+	message = replacetext(message, "cult", "super magical group")
+	message = replacetext(message, "apologies", "gomenasai")
+	message = replacetext(message, "sexual", "l-lewd...")
```

Comment:
This would change "I'm gonna have sexual thoughts about that" to "I'm gonna have l-lewd... thoughts about that"

maybe remove the suspension point, and have the stutter on a prob chance?

Reply:
Your wish is my command